gcloud/bigquery/job.py

```diff
@@ -0,0 +1,702 @@
+
+        :rtype: list of mappings, or ``NoneType``
+        :returns: the error information (None until set from the server).
+        """
+        status = self._properties.get('status')
+        if status is not None:
+            return status['errors']
+
+    @property
+    def state(self):
+        """Status of the job.
+
+        :rtype: string, or ``NoneType``
+        :returns: the state (None until set from the server).
+        """
+        status = self._properties.get('status')
+        if status is not None:
+            return status['state']
+
+    @property
+    def allow_jagged_rows(self):
+        """Allow rows with missing trailing commas for optional fields.
+
+        :rtype: boolean, or ``NoneType``
+        :returns: The value as set by the user, or None (the default).
+        """
+        return self._configuration._allow_jagged_rows
+
+    @allow_jagged_rows.setter
+    def allow_jagged_rows(self, value):
+        """Update allow_jagged_rows.
+
+        :type value: boolean
+        :param value: new allow_jagged_rows
+
+        :raises: ValueError for invalid value types.
+        """
+        if not isinstance(value, bool):
+            raise ValueError("Pass a boolean")
+        self._configuration._allow_jagged_rows = value
+
+    @allow_jagged_rows.deleter
+    def allow_jagged_rows(self):
+        """Delete allow_jagged_rows."""
+        del self._configuration._allow_jagged_rows
+
+    @property
+    def allow_quoted_newlines(self):
+        """Allow rows with quoted newlines.
+
+        :rtype: boolean, or ``NoneType``
+        :returns: The value as set by the user, or None (the default).
+        """
+        return self._configuration._allow_quoted_newlines
+
+    @allow_quoted_newlines.setter
+    def allow_quoted_newlines(self, value):
+        """Update allow_quoted_newlines.
+
+        :type value: boolean
+        :param value: new allow_quoted_newlines
+
+        :raises: ValueError for invalid value types.
+        """
+        if not isinstance(value, bool):
+            raise ValueError("Pass a boolean")
+        self._configuration._allow_quoted_newlines = value
+
+    @allow_quoted_newlines.deleter
+    def allow_quoted_newlines(self):
+        """Delete allow_quoted_newlines."""
+        del self._configuration._allow_quoted_newlines
+
+    @property
+    def create_disposition(self):
+        """Handling for missing destination table.
+
+        :rtype: string, or ``NoneType``
+        :returns: The value as set by the user, or None (the default).
+        """
+        return self._configuration._create_disposition
+
+    @create_disposition.setter
+    def create_disposition(self, value):
+        """Update create_disposition.
+
+        :type value: boolean
+        :param value: new create_disposition: one of "CREATE_IF_NEEDED" or
+                      "CREATE_NEVER"
+
+        :raises: ValueError for invalid value.
+        """
+        if value not in ('CREATE_IF_NEEDED', 'CREATE_NEVER'):
```

Comment:
4ed658862c49242eb9dd1cec2c62a073333170ee adds some quasi-enum classes to support passing known-good values by name, rather than as typo-able string literals.  E.g.:

```python
    from gcloud.bigquery.job import CreateDisposition
    job.create_disposition = CreateDisposition.CREATE_NEVER
```